src/openqdc/datasets/gdml.py

```diff
@@ -13,9 +13,11 @@ class GDML(BaseDataset):
     atomic_energies = np.zeros((MAX_ATOMIC_NUMBER,), dtype=np.float32)
 
     __energy_methods__ = [
-        "ccsd",
-        "ccsd(t)",
-        "pbe-ts",
+        "ccsd/cc-pvdz",
+        "ccsd(t)/cc-pvdz",
+        # "pbe+mbd/light", #MD22
```

Comment:
We are not using those methods should we?

Reply:
It depends if by GDML we are talking about MD17, MD22 or both. Both of them are calculated from AIMD with FHI-aims with All Electron DFT. MD17 uses the pbe functional with vdw-ts, while MD22 uses pbe+mbd with different settings per system. 
I'm going to calculate the isolated atoms energy for  ccsd/cc-pvdz, ccsd(t)/cc-pvdz and I'm creating as we speak the pseudopotential for the pbe functional for the pbe+vdw-ts level of theory on DFT PAW. We would have issue on the many body dispersion as I can't use the correct convergence light/tight so I think just using the pbe+vdw-ts should be good enough